### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**

### CODE REVIEW

**Review by tetsuo-cpp — APPROVED** [review]
LGTM @kianmeng! Once you sign the CLA, I can go ahead and merge this.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**
